dask/bag/core.py

```diff
@@ -675,9 +675,9 @@ def random_sample(self, prob, random_state=None):
         >>> import dask.bag as db
         >>> b = db.from_sequence(range(5))
         >>> list(b.random_sample(0.5, 43))
-        [0, 3, 4]
+        [0, 1, 4]
         >>> list(b.random_sample(0.5, 43))
-        [0, 3, 4]
+        [0, 1, 4]
```

Comment:
Annoyingly this doctest will fail if a user hasn't installed numpy. I don't think it's a big problem.